src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,194 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceTests {
```

Comment:
The class name `RegistrationServiceTests` does not follow the Java naming convention for test classes. According to the task's convention, it should be `RegistrationServiceImplTest` to reflect the class being tested.

Reply:
changed